hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/portable/PortableGenericRecord.java

```diff
@@ -16,17 +16,20 @@
 
 package com.hazelcast.internal.serialization.impl.portable;
 
+import com.hazelcast.internal.serialization.impl.AbstractGenericRecord;
 import com.hazelcast.nio.serialization.ClassDefinition;
 import com.hazelcast.nio.serialization.FieldDefinition;
 import com.hazelcast.nio.serialization.FieldType;
 import com.hazelcast.nio.serialization.GenericRecord;
 import com.hazelcast.nio.serialization.HazelcastSerializationException;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.jetbrains.annotations.NotNull;
```

Comment:
I wonder if is there a way to tell IDE not to set this by default

Reply:
It would be even nicer to have a linter rule to fail code style validation if `org.jetbrains.annotations.NotNull` is in use. It seems to be a common pitfall.